src/story.ts

```diff
@@ -120,6 +120,16 @@ export type StoryContextForLoaders<
   TArgs = Args
 > = StoryContextForEnhancers<TRenderer, TArgs> &
   Required<StoryContextUpdate<TArgs>> & {
+    /**
+     * The unmappedArgs are the args that are passed over the channel
+     * and defined in the story definition. They must be serializable.
+     */
+    unmappedArgs: TArgs;
+    /**
+     * The args that are passed on the context are mapped by arg mapping
+     * and can be filtered by conditional arg filters.
+     */
+    args: Args;
```

Comment:
So we kind of want to override the args from TArgs to Args here right?
Because they are defined as TArgs in StoryContextUpdate.

Reply:
I guess they should be `Args` in `StoryContextUpdate` too, after all.